packages/eslint-plugin/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "declaration": false,
+    "outDir": "./dist",
```

Comment:
wouldn't it make more sense to output to `lib/` still? the js sources used to be there, so the js output could be too.

Reply:
the benefit of having the `dist` folder is that we can have a post-build step to copy the other required build files across (README.md, package.json, etc), and then you can do `npm publish dist`, and it'll publish the dist folder.